src/js/jsx/search/SearchBar.jsx

```diff
@@ -515,7 +531,7 @@ define(function (require, exports, module) {
                 var lastFilter = filter[filter.length - 1],
                     categoryString = searchStrings.CATEGORIES[lastFilter],
                     category = categoryString ?
-                        categoryString.toLowerCase() : this.state.safeFilterNameMap[lastFilter];
+                        categoryString : this.state.safeFilterNameMap[lastFilter];
```

Comment:
This conversion to lower case was inconsistent with how the strings in the search bar were being presented. 

In the dropdown, the options were capitalized.
![](https://www.dropbox.com/s/cnvlym5r8syy0hf/Screenshot%202016-01-06%2015.18.34.png?raw=1)

Upon selection, the placeholder text would become lower case. 
![](https://www.dropbox.com/s/airj8psgvzyw4io/Screenshot%202016-01-06%2015.18.56.png?raw=1)

I talked to Seth about this and he said removing the lower case upon selection makes sense.